src/main/java/no/ndla/taxonomy/service/DomainEntityHelperServiceImpl.java

```diff
@@ -221,37 +222,47 @@ private Optional<DomainEntity> updateNodeConnection(NodeConnection nodeConnectio
         NodeConnection existing = nodeConnectionRepository.findByPublicId(nodeConnection.getPublicId());
         if (existing == null) {
             // Use correct objects when copying
-            nodeConnection.setParent(nodeRepository.findByPublicId(nodeConnection.getParent().get().getPublicId()));
-            nodeConnection.setChild(nodeRepository.findByPublicId(nodeConnection.getChild().get().getPublicId()));
+            Node parent = nodeRepository.findByPublicId(nodeConnection.getParent().get().getPublicId());
```

Comment:
Gir det egentlig mening at en `getParent` og `getChild` på en `NodeConnection` er `Optional`?

Hadde egentlig tenkt å spørre om vi kunne vite at `.get`'en her alltid stemte, men jeg kunne ikke skjønne hvorfor man ville ha en NodeConnection uten barn eller forelder.

Reply:
Nei, ikkje egentlig. Kan sjå kva som skal til for å endre.